homeassistant/components/camera/push.py

```diff
@@ -45,7 +45,9 @@
                           config[CONF_BUFFER_SIZE],
                           config[CONF_TIMEOUT])]
 
-    hass.http.register_view(CameraPushReceiver(cameras,
+    component = hass.data.get(DOMAIN)
```

Comment:
You're not allowed to do this. You have to keep track of your own entities yourself. This is an internal thing of the camera component.